frontend/src/views/ComplianceReports/EditViewComplianceReport.jsx

```diff
@@ -92,9 +92,9 @@ export const EditViewComplianceReport = () => {
     complianceReportId
   )
 
-  const currentStatus = reportData?.data?.currentStatus?.status
+  const currentStatus = reportData?.report.currentStatus?.status
```

Comment:
reportData?.report seems to be unnecessary, can you please recheck this as it is failing
image